src/components/Canary/renderers.js

```diff
@@ -94,15 +94,17 @@ export function Percentage({ val, upper, lower }) {
   );
 }
 
-export function Title({ icon, title }) {
+export function Title({ icon, title, isDeleted }) {
   return (
     <>
       {icon && (
         <span className="w-6 flex-shrink-0 mr-2">
           <Icon name={icon} className="inline h-6" />
         </span>
       )}
-      <span className="text-sm">{title}</span>
+      <span className="text-sm">
```

Comment:
I think ideally, we do something as we did with config items -> gray the whole row, and instead of the text deleted, we show the trash icon.

Reply:
@mainawycliffe what config items are in gray? Can you point me to the code?

Reply:
@adityathebe sorry, missed this, this is the function that grays out the whole row for config list -> https://github.com/flanksource/flanksource-ui/blob/1233-topology-navigation-progress-bar/src/components/ConfigList/index.tsx#L117 and it's passed to the DataTable at this point https://github.com/flanksource/flanksource-ui/blob/9ad08e4fbe9fe4af7530c4d205cb049cf2ee7e5d/src/components/ConfigList/index.tsx#L156C1-L157C1

Reply:
https://github.com/flanksource/flanksource-ui/pull/1267#issuecomment-1611111728